components/cd-dropdown/cd-dropdown.js

```diff
@@ -19,11 +22,37 @@
       this.updateDrupalTogglableMenus();
     },
 
+    /**
+     * Get the toggler element form the togglable element.
+     */
+    getTogglerElement: function (element) {
+      var id = this.getToggableId(element);
+      return document.querySelector('[data-cd-toggler][aria-controls="' + id + '"]');
+    },
+
+    /**
+     * Get the togglable element form the toggler element.
+     */
+    getTogglableElement: function (toggler) {
```

Comment:
Ah, dammit, this is the kind of typo that will stay forever. If we could change it I would go with `togglable` which apparently is more correct according to a quick dictionary search but that would mean a lot of code changes across the sites because of the `data-cd-toggable` attribute...

We can't go with `toggable` everywhere either due to the `updateDrupalTogglableMenus` function being called in cd-menu.js and possibly other places and would also require changes across the sites.

I would propose to fix the spelling of the internal functions to use `togglable` at least and make a note of the spelling issue of the `data-cd-toggable`. @rupl @left23 What do you think?

Reply:
No opinions only sympathy :) And I also laughed. I think if we did decided to change it to whatever is deemed better, the updating of all the instances would be feasible enough as it's findable and finite. Is there a way to do it in a few phases and keep it backwards compatible for a while? Is that worse?!

Reply:
Ha, yeah... I never brought it up til now because I also didn't want to deal with cleaning individual sites up :)

Ok, so internal functions will be updated to add the `l`. I'll take over the work and do a release when I'm done so you can go back to UNO stuff and try out the upgrade tomorrow.

Reply:
I was curious about how hard we'd have to work to correct all the instances and there were too many pages to quickly categorize the need for action:

https://github.com/search?q=org%3AUN-OCHA+data-cd-toggable%3D%22&type=code

The JS functions themselves are replicated in a couple places, but seems they are manually copied and reused quite seldom. I believe all instances of `setToggable` are inside statically-generated JS aggregates OR snowflake projects like CD WP, HID, publii, etc. Where that one function lives, the other functions do too; in the snowflakes, we're normally wholesale replacing files when upgrading them to new versions of CD.

https://github.com/search?q=org%3AUN-OCHA%20setToggable&type=code

Anyway, I renamed the internal functions. I don't even see the need to note the misspelling for the data-attribute because it's always been that way and all our code examples are consistent over time, if not in spelling. So any templates that get copied are also tending to work out of the box, and few will find reason to pay attention to the fine print.